Source/AppAuthTV/OIDTVTokenRequest.h

```diff
@@ -0,0 +1,79 @@
+/*! @file OIDTVTokenRequest.h
+   @brief AppAuth iOS SDK
+   @copyright
+       Copyright 2020 Google Inc.
+   @copydetails
+       Licensed under the Apache License, Version 2.0 (the "License");
+       you may not use this file except in compliance with the License.
+       You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+       Unless required by applicable law or agreed to in writing, software
+       distributed under the License is distributed on an "AS IS" BASIS,
+       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+       See the License for the specific language governing permissions and
+       limitations under the License.
+*/
+
+#import "OIDTokenRequest.h"
+
+#import <Foundation/Foundation.h>
+
+@class OIDServiceConfiguration;
+@class OIDTVServiceConfiguration;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OIDTVTokenRequest : OIDTokenRequest
+
+/*! @brief The device code received from the authorization server.
+    @remarks device_code
+    @see https://tools.ietf.org/html/rfc8628#section-3.4
+ */
+@property(nonatomic, readonly, nullable) NSString *deviceCode;
```

Comment:
Is there a situation that this can be nil? If not we should remove `nullable`.

Reply:
It is required, will remove nullable. (https://tools.ietf.org/html/rfc8628#section-3.4)

Reply:
Interestingly, I noticed that in the parent class authorizationCode is nullable, but it is required per the rfc so I'll remove nullable